db/migrate/20160913023501_create_items.rb

```diff
@@ -0,0 +1,11 @@
+class CreateItems < ActiveRecord::Migration[5.0]
+  def change
+    create_table :items do |t|
+      t.string :title
+      t.string :description
+      t.integer :price
+      t.string :image_path
+      t.timestamp null: false
```

Comment:
I think this line needs to be t.timestampS. The `created_at` and `updated_at` doesn't look like they got created in the schema.